components/speedreader/resources/panel/components/toolbar/index.tsx

```diff
@@ -128,7 +147,7 @@ function Toolbar() {
         onClick={handleMainButtonClick.bind(this)}
       />
       {(activeButton === MainButtonType.None || activeButton === MainButtonType.Tune) &&
-       (<ReaderModeControl onClose={handleClose.bind(this)} />)
+        (<ReaderModeControl onClose={handleClose.bind(this)} />)
```

Comment:
nit: (as this line is being changed)

```suggestion
        (<ReaderModeControl onClose={() => handleClose()} />)
```